new_core_concepts/modules/ROOT/pages/typeql/query-variables-patterns.adoc

```diff
@@ -79,7 +79,7 @@ so that every constraint in the pattern holds true in the database,
 and every subpattern is also satisfied.
 
 Each such mapping from variables to concepts is an [[definition-answer]]*answer* to the pattern.
-For the query above, each answer maps `$e` to an entity, and `$a` to an attribute owned by that type.
+For the query above, each answer maps `$e` to an entity, and `$e` to a relation linking that entity.
```

Comment:
Remove this line please. It was meant for the example above and no longer serves a purpose here.